src/app/game/game.component.html

```diff
@@ -1,16 +1 @@
-<mat-grid-list [cols]="2">
-  <mat-grid-tile>
-    <app-game-table
-      [dataSource]="bot1Stats"
-      [botName]="bot1Name"
-      class="allTitle"
-    ></app-game-table>
-  </mat-grid-tile>
-  <mat-grid-tile>
-    <app-game-table
-      [dataSource]="bot2Stats"
-      [botName]="bot2Name"
-      class="allTitle"
-    ></app-game-table>
-  </mat-grid-tile>
-</mat-grid-list>
+<app-game-table></app-game-table>
```

Comment:
Add empty line at the end of file.
Also: Do you really need this file? There is only one line with empty tag...

Reply:
Game component will soon contain more other components to present game statistics.